falcon/response.py

```diff
@@ -918,7 +922,7 @@ def append_link(
             _headers['link'] = value
 
     # NOTE(kgriffs): Alias deprecated as of 3.0
-    add_link = append_link
+    add_link = deprecated(_ADD_LINK_DEPRECATED_MSG, method_name='add_link')(append_link)
```

Comment:
Without overriding the `method_name`, there was a sneaky issue with this approach -- we were actually warning that `append_link(...)` was deprecated, and asking to use `append_link()` instead :man_facepalming:

Reply:
nice catch!